client/i18n/locales/english/intro.json

```diff
@@ -3412,6 +3412,12 @@
           "In this lab, you will create a function that removes all falsy values from an array."
         ]
       },
+      "lab-title-case-converter": {
```

Comment:
following the order on the sheet, can you move this above the falsy remover lab?